.github/workflows/pkgci_regression_test_cpu.yml

```diff
@@ -58,20 +58,32 @@ jobs:
             experimental/regression_suite
 
       # Out of tree tests
-      - name: Checking out external TestSuite repository
+      - name: Check out external TestSuite repository
         uses: actions/checkout@8f4b7f84864484a7bf31766abe9204da3cbe65b3 # v3.5.0
         with:
           repository: nod-ai/SHARK-TestSuite
-          ref: 46ab63b271fddd78e071c0590e9a073f6282b005
+          ref: e266c91e03734fe3148f269b02ff4b9e4bfd5df9
           path: SHARK-TestSuite
           submodules: false
-      - name: Installing external TestSuite Python requirements
+          lfs: true
```

Comment:
Need to install git-lfs somehow

https://github.com/openxla/iree/actions/runs/8303455114/job/22727749815?pr=16801#step:8:30
`Error: Unable to locate executable file: git-lfs`

Could include it on the runner... whichever this is using
```
    runs-on:
      - self-hosted # must come first
      - runner-group=${{ github.event_name == 'pull_request' && 'presubmit' || 'postsubmit' }}
      - environment=prod
      - cpu
      - os-family=Linux
```

Or just run `git lfs install`